src/main/java/ladder/view/ResultView.java

```diff
@@ -0,0 +1,28 @@
+package ladder.view;
+
+import ladder.model.Line;
+import ladder.model.Player;
+
+import java.util.List;
+
+public class ResultView {
+
+    public static void printResult(List<Player> players, List<Line> lineList) {
```

Comment:
```suggestion
    public static void printResult(List<Player> players, List<Line> lines) {
```
lineList와 같이 변수명에 타입을 지정한다면 어떤 문제점이 있을지 고민해보세요 😄

Reply:
list아닌 다른 타임을 사용하게 된다면 의미가 헷갈리게 도리 것 같습니다.